app/models/container_quota_item.rb

```diff
@@ -1,3 +1,29 @@
 class ContainerQuotaItem < ApplicationRecord
   belongs_to :container_quota
+  BINARY_SUFFICES = ["Ki", "Mi", "Gi", "Ti", "Pi", "Ei"].freeze
+  SI_SUFFICES = ["K", "M", "G", "T", "P", "E"].freeze
+
+  def normalize_field_for_chargeback(field_sym)
+    raw_field = send(field_sym)
+    case resource
+      when "cpu" # 1000m = 1 core - convert to cores
+        raw_field.end_with?("m") ?  raw_field.chomp("m").to_i / 1000 : raw_field.to_i
+      when "memory" # could be "Ki", "Mi", "Gi", "Ti", "Pi", "Ei" or in bytes - convert to bytes
```

Comment:
maybe i got the term 'raw' wrong. its not really raw data, its text. its just that it could be in a lot of formats.